Monika After Story/game/script-topics.rpy

```diff
@@ -13165,3 +13166,30 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_songwriting:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3eua "I've tried myself, and it's pretty fun. It takes a bit of time to do, but the end product is definitely worth it!"
```

Comment:
I would add a comma after "tried".

Also, I'm pretty sure anyone playing MAS knows that Monika has written a song before